### PR TITLE
163518693 Users Follow Authors

### PR DESCRIPTION
**Pull request by abayo-luc**
#### What does this PR do?
Enable user to follow auther
#### Description of Task to be completed?
```POST /api/v1/profiles/:username/follow```
```DELETE /api/v1/profiles/:username/follow```
```GET /api/v1/profiles/:username/followers```
```GET /api/v1/profiles/:username/followings```
#### How should this be manually tested?
-Clone the repo and checkout `ft-user-follow-each-other-163518693` branch
- Using any `API client tool(usually POSTMAN)`,
- Create the user and login to get the `access-token`
- Make request to the above endpoint with existing ```username```
- Make sure to pass the token under `request headers`. It should be {`Authorization`: `Bearer <access-token>`} KEY, VALUE PAIR
#### Any background context you want to provide?
before running anything, ensure packages are updated ```yarn``` and no pending migration ```sequelize db:migrate```
#### What are the relevant pivotal tracker stories?
163518693

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -46,50 +46,8 @@ if (!isProduction) {
...
-      message: err.message,
-      error: {}
-    }
-  });
-});
-
-// finally, let's start our server...
 const server = app.listen(process.env.PORT || 3000, () => {
-  // eslint-disable-next-line no-console
   console.log(`Listening on port ${server.address().port}`);
```
Comment (YvesIraguha) [inline index.js:50]: Can't this `console.log()` be removed?

File: tests/users/follower.test.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+describe("User following", () => {
+  before(async () => {
+    const user1 = await chai
+      .request(app)
+      .post("/api/v1/users")
+      .send({
+        ...dummyUser
+      });
+    const user2 = await await chai
```
Comment (YvesIraguha) [inline tests/users/follower.test.js]: I think this is a repetition error `await await` correct it.

**Review by YvesIraguha — APPROVED** [review]
Correct the repetition error of `await await` and remove `console.log()` in the index.js file.

**Review by Musigwa-zz — COMMENTED** [review]
I think if there would be a way of reducing the number of commits and remain with the latest few commits, it will be better and easier for the next rebasing work.

File: index.js (hunk 1 above)
Comment (abayo-luc) [reply index.js:50]: @YvesIraguha we decided to let it there in the meanwhile.

**Review by Musigwa-zz — APPROVED** [review]

**Pull request merged**